docs/demo.rst

```diff
@@ -0,0 +1,22 @@
+A Simple Demo for GluonNLP
+==========================
+
+Introduction
+------------
+
+This demo is using MMS Server
```

Comment:
Sentiment Analysis predicts whether an input is positive or negative. The model is based on BERT base, and are trained on the binary classification setting of the Stanford Sentiment Treebank. It achieves about 87% and 93.4% accuracy on the test set.